### ISSUE TITLE
bash does not work

### ISSUE THREAD

**Issue by kiyochin9** [git-for-windows/git#4578]
- [ ] I was not able to find an [open](https://github.com/git-for-windows/git/issues?q=is%3Aopen) or [closed](https://github.com/git-for-windows/git/issues?q=is%3Aclosed) issue matching what I'm seeing

### Setup

 - Which version of Git for Windows are you using? Is it 32-bit or 64-bit?

```
$ git --version --build-options

** insert your machine's response here **
```

 - Which version of Windows are you running? Vista, 7, 8, 10? Is it 32-bit or 64-bit?

```
$ cmd.exe /c ver

** insert your machine's response here **
```

 - What options did you set as part of the installation? Or did you choose the
   defaults?

```
# One of the following:
> type "C:\Program Files\Git\etc\install-options.txt"
> type "C:\Program Files (x86)\Git\etc\install-options.txt"
> type "%USERPROFILE%\AppData\Local\Programs\Git\etc\install-options.txt"
> type "$env:USERPROFILE\AppData\Local\Programs\Git\etc\install-options.txt"
$ cat /etc/install-options.txt

** insert your machine's response here **
```

 - Any other interesting things about your environment that might be related
   to the issue you're seeing?

** insert your response here **

### Details

 - Which terminal/shell are you running Git from? e.g Bash/CMD/PowerShell/other

** insert your response here **

 - What commands did you run to trigger this issue? If you can provide a
   [Minimal, Complete, and Verifiable example](http://stackoverflow.com/help/mcve)
   this will help us understand the issue.

```
** insert your commands here **
```
 - What did you expect to occur after running these commands?

** insert here **

 - What actually happened instead?

** insert here **

 - If the problem was occurring with a specific repository, can you provide the
   URL to that repository to help us with testing?

** insert URL here **

**Comment by kiyochin9** [comment]
Hi,
I am using GIT on Windows 10 machine. Issue is that Git-Bash stopped working with this comment at the header portion of the command window, "Failed to run '/usr/bin/bash': No such file or directory". It was working until 2 weeks ago. I reinstalled the latest GIT, but the symptom is the same.
However, I can run git on the Window PowerShell. From a local branch, if I fetch a remote branch, it says this, "fatal: 'remote-https' appears to be a git command, but we were not able to execute it. Maybe git-remote-https is broken?".
I can pull the same remote branch into a new local branch, but fetch and push does not work as described above. 
Here is information about my system and GIT. 
1) Windows 10 running on Microsoft Surface Book 3
2) Git version: git --version
git version 2.42.0.windows.2
3)  cmd.exe /c ver
Microsoft Windows [Version 10.0.19045.3393]
4) Install option
Editor Option: VIM
Custom Editor Path: 
Default Branch Option:  
Path Option: Cmd
SSH Option: OpenSSH
Tortoise Option: false
CURL Option: OpenSSL
CRLF Option: CRLFAlways
Bash Terminal Option: MinTTY
Git Pull Behavior Option: Merge
Use Credential Manager: Enabled
Performance Tweaks FSCache: Enabled
Enable Symlinks: Disabled
Enable Pseudo Console Support: Disabled
Enable FSMonitor: Disabled
5) Terminal
I was using the Bash before. But I am now using Windows PowerShell due to the issue. But GIT fetch and pull are still not working as described above.
6) Repository: I tried three other remote repositories. but all them failed with the same symptom. 

Your prompt suggestion would be greatly appreciated.
